.travis.yml

```diff
@@ -5,7 +5,7 @@ env:
 - ACTION="run lint"
 - ACTION="run dist" # ensures building dist files doesn't break
 - ACTION="run cs-check"
-- SERVER=6.0.1 ACTION=test
+- SERVER=8.1.2 ACTION=test
```

Comment:
some old version on purpose?

Reply:
I don't understand. 8.1.2 *is* the old version on purpose. 7.0.0 is the oldest one that supports `accounts`, but I didn't see a reason to keep anything from that era around.